tests/assertions/ocp4/rhcos4-e8-4.12.yml

```diff
@@ -3,11 +3,11 @@ rule_results:
     default_result: PASS
     result_after_remediation: PASS
   e2e-e8-master-audit-rules-dac-modification-chmod:
-    default_result: FAIL
-    result_after_remediation: PASS
+    default_result: FAIL OR NOT-APPLICABLE
```

Comment:
We might still need to lower case these (`or`).

Reply:
opps I didn't include the commits